### PR TITLE
updated isArray()

### PR DESCRIPTION
**Pull request by saurabhkanswal**
i have added some more information with extra resource link.

### CODE REVIEW

**Comment by saurabhkanswal** [comment]
why PR is showing not eligible in hacktoberfest dashboard.

File: 1 Foundations/isArray.js [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+// syntax: Array.isArray(value)
+
+// Parameters:
+//            value means the value to be checked
+
 (function () {
     console.log(Array.isArray(arguments))
 })()
+
+// Return value: true if given parameter is an array otherwise it return false
```
Comment (MadaZZ) [inline 1 Foundations/isArray.js:15]: The explanations look good! 👍🏻
Please look into [SampleJS.md](https://github.com/MadaZZ/JS-Concepts/blob/master/SampleJS.md) for file format

**Review by MadaZZ — CHANGES_REQUESTED** [review]
Please change the format as requested. :)

File: 1 Foundations/isArray.js [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+
+
+
+// Return value: true if given parameter is an array otherwise it return false
 (function () {
-    console.log(Array.isArray(arguments))
+    console.log(Array.isArray(arguments)) //checking that arguments is array or not.
 })()
+
+
```
Comment (MadaZZ) [inline 1 Foundations/isArray.js:17]: Do not think this needs a standalone file!

**Review by MadaZZ — COMMENTED** [review]
This can be written as a part of some other file. It is not something that needs a standalone program.

**Pull request closed without merging**